pkg/client/clientset/versioned/typed/cassandra.rook.io/v1alpha1/cluster.go

```diff
@@ -75,23 +77,33 @@ func (c *clusters) Get(name string, options v1.GetOptions) (result *v1alpha1.Clu
 
 // List takes label and field selectors, and returns the list of Clusters that match those selectors.
 func (c *clusters) List(opts v1.ListOptions) (result *v1alpha1.ClusterList, err error) {
+	var timeout time.Duration
+	if opts.TimeoutSeconds != nil {
+		timeout = time.Duration(*opts.TimeoutSeconds) * time.Second
+	}
 	result = &v1alpha1.ClusterList{}
 	err = c.client.Get().
 		Namespace(c.ns).
 		Resource("clusters").
 		VersionedParams(&opts, scheme.ParameterCodec).
```

Comment:
The opts here no longer observes the `v1.ListOptions` `TimeoutSeconds` field?

Reply:
This is generated code, but the timeout is used on the next line, right?